lib/rest_in_peace/template_sanitizer.rb

```diff
@@ -16,10 +17,14 @@ def url
       return @url if @url
       @url = @url_template.dup
       tokens.each do |token|
-        param = @params.delete(token.to_sym)
```

Comment:
@luxflux I think this case is covered  [here](https://github.com/ninech/REST-in-Peace/blob/master/spec/rest_in_peace/template_sanitizer_spec.rb#L15-L19)